apps/condo/domains/acquiring/utils/taskSchema/index.js

```diff
@@ -151,7 +151,8 @@ async function getReceiptsForServiceConsumer (context, date, serviceConsumer, bi
 
     const periodDate = dayjs(date)
     const period = periodDate.format('YYYY-MM-01')
-    const previousPeriod = periodDate.startOf('month').subtract(1, 'days').format('YYYY-MM-01')
+    const monthBeforePeriod = periodDate.startOf('month').subtract(1, 'month').format('YYYY-MM-01')
+    const twoMonthBeforePeriod = periodDate.startOf('month').subtract(2, 'month').format('YYYY-MM-01')
```

Comment:
What the difference between `YYYY-MM-01` and `YYYY-MM-DD` date formats?

Reply:
Since I'm looking for receipts in certain periods, I have to follow billingReceipt.period field format. The format is: YYYY-MM-01, that is literally the first day of the month. I prefer to use exact the same format, like in billingReceipt.period